recipes/python-box/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "python-box" %}
+{% set version = "3.1.1" %}
+{% set sha256 = "b9192e2a2c7f4535317dfb25b7ae26d8d66f57d29b47d43054aab1ac5df53c07" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pytest-runner
+    - pyyaml
```

Comment:
Are all of these build requirements?

Reply:
pyyaml - can be removed, but pytest-runner is required as the tests are executed after the build process to validate the installation. @CJ-Wright should I also add you to the list of maintainers? https://github.com/cdgriffith/Box/pull/50#issuecomment-377513167